### ISSUE TITLE
23 x GPTs

### ISSUE THREAD

**Issue by orrenp** [polywock/Awesome-GPTs#22]
https://chat.openai.com/g/g-rJ8laM2qH-copywriter
https://chat.openai.com/g/g-5rBXWMTTN-chat-with-many-urls
https://chat.openai.com/g/g-F8w8xIKqY-nft
https://chat.openai.com/g/g-VKcQOkPTV-bitcoin-sentiment
https://chat.openai.com/g/g-sycM3WZAT-aus-petrol-prices
https://chat.openai.com/g/g-MibXZy4B6-aus-surf-report
https://chat.openai.com/g/g-F1EMXb3ZP-sa-speed-cameras
https://chat.openai.com/g/g-ZNvavsN3l-currency-converter
https://chat.openai.com/g/g-BAo0qPpm8-ai-news-roundup
https://chat.openai.com/g/g-a61upqaQQ-interest-rates
https://chat.openai.com/g/g-3UyUzTfnG-p-e-for-nfts
https://chat.openai.com/g/g-5Xc1AzVCC-nft-news-roundup
https://chat.openai.com/g/g-NsJAyoYRH-what-would-muhammad-do
https://chat.openai.com/g/g-gqozZuI8N-south-australia-speed-camera-law
https://chat.openai.com/g/g-j6r5TmGrB-logo-creator
https://chat.openai.com/g/g-MSLbCKHy3-what-would-jesus-do
https://chat.openai.com/g/g-KQwBRMM3Q-eco-luxe-guide
https://chat.openai.com/g/g-lPVKcntPd-printpal
https://chat.openai.com/g/g-AsuRPvdUm-euromobile-assistant
https://chat.openai.com/g/g-v1dZCK7fC-what-would-machiavelli-do
https://chat.openai.com/g/g-wrUEOZ3WY-what-would-carl-von-clausewitz-say
https://chat.openai.com/g/g-jkofWqFvS-what-would-sun-tzu-do
https://chat.openai.com/g/g-rbV6ySJI0-website-ui-ux-review-suggestions

**Comment by polywock** [comment]
You've been busy! I hate too make you choose, but any five you're particularly proud of?

**Comment by orrenp** [comment]
1.
AI News Roundup
  2.
Bitcoin Sentiment
  3.
TL;DR
  4.
Content Rewriter
  5.
SEO

________________________________

**Comment by polywock** [comment]
Congratulations on making it to the official store. SEO is 7th on the Research/Analysis section.
